config/routes.rb

```diff
@@ -4,6 +4,8 @@
   default_url_options Rails.application.config.action_mailer.default_url_options
   concern :exportable, Blacklight::Routes::Exportable.new
 
+  get 'filemanager/index'
```

Comment:
What is this for? If I try to access this path, I get an error about `FilemanagerController` not being defined.